lib/glific/third_party/sheets/sheet.ex

```diff
@@ -51,6 +51,24 @@ defmodule Glific.Sheets.Sheet do
     sheet
     |> cast(attrs, @required_fields ++ @optional_fields)
     |> validate_required(@required_fields)
+    |> validate_url()
     |> unique_constraint([:label, :url, :organization_id])
   end
+
+  @spec validate_url(Ecto.Changeset.t()) :: Ecto.Changeset.t()
+  defp validate_url(%{changes: changes} = changeset) when not is_nil(changes.url) do
+    url = changeset.changes[:url]
+
+    if Glific.URI.cast(url) == :ok && String.ends_with?(url, "output=csv") do
```

Comment:
This is a parameter so it's not necessary that it will always end with "output=csv"

```suggestion
    if Glific.URI.cast(url) == :ok && String.contains?(url, "output=csv") do
```